liferay-echo/liferay-portal
# LPS-198338 Adds readonly attribut to infoField

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-198338)

Adds readOnly attribute to infoField, and adapt use cases. This PR is sent on top of the awesome [PR](https://github.com/liferay-echo/liferay-portal/pull/13925) by @veroglez to make it easier to test.

## Change summary:

* Just as the title says


## Test Scenario

* Activate the FF LPS-183727
* Go to Apps Menu > Control Panel > Objects
* Create a new Object, and add a Field to it
* Edit the field, and change so that readOnly is set to true
* Make the object either system or group object
* Go to any site and create a new page
* Add a form container, and select the object that was created
* All the fields will be displayed
* Publish the page and navigate to it
* You will see that the input is in readOnly mode
